run_integration_tests.go

```diff
@@ -0,0 +1,189 @@
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+type CIEnvironment interface {
```

Comment:
I think the Go naming convention would prefer this to be called "CiEnvironment"? (lowercase i)